eloquentjs/15_1_balloon/index.js

```diff
@@ -0,0 +1,40 @@
+let balloon = document.getElementById("balloon");
+
+let balloonSize = 1;
+let explosionThreshold = 18;
+
+let timeout;
+
+let balloonHandler = event => {
+    if (event.key === 'ArrowUp') {
+        clearTimeout(timeout);
+
+        timeout = setTimeout(() =>{
```

Comment:
I don't think you should use setTimeout here. It would be better to avoid that complexity.